metafacture-runner/src/main/dist/examples/count/subjects/references.morph.flux

```diff
@@ -13,7 +13,6 @@ decode-pica|
 morph(FLUX_DIR + "references.xml")|
 stream-to-triples|
 count-triples(countBy="object")|
-
```

Comment:
This empty line is still in the Fix example.

Reply:
Should be fixed.

Reply:
I've fixed it in b0c99c0.